### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by djmtype** [bitgapp/eqMac#601] [General]
The official site states Pro users will have access to customizable UI. Does that mean the general user interface will be refreshed? 

As a personal preference the UI looks a bit dated and/or not well integrated with the OS. Skeumorphic design can still be appealing, but this still reminds me too much of an old soundboard.

**Comment by nodeful** [comment]
Hey there, thanks for sharing your views on the design. I generally agree with you, the design is up for a refresh for sure, I also want to get it to look native to macOS Monterey and it's definitely possible. Right now however I'm focusing my efforts on the functionality, bug fixing and business side of things for the nearest future.
To answer your questions about the planned Customisable UI feature, this mostly means that non-Pro users get to choose between light/dark mode where's Pro users can precisely select the color scheme themselves (background, accent, tint, text color etc) :)

**Comment by nodeful** [comment]
I've implemented Light/Dark theme with a little bit of a redesign. Should be available in app version v1.3.0 and higher if you have OTA Updates enabled.
![image](https://user-images.githubusercontent.com/8472525/149200591-03abba15-6190-413a-aa82-422ca6e4541a.png)
![image](https://user-images.githubusercontent.com/8472525/149200620-9b00116b-e3d5-4017-b3b1-cc7f2f1ace3d.png)

**Comment by mustevenplay** [comment]
Looks great!!

**Comment by nodeful** [comment]
Thank you :) There will be more tweaks over time:
1. Probably will have just 1 Knob style instead of current 3 styles.
2. Going to remove the Speaker vent on the bottom right and replace it with a button to adjust Output settings once those are implemented.
3. Going to allow Pro customers to change Background, Foreground (text etc.) and Accent colors.
4. Make UI modules draggable and arrangeable into multiple columns
Feel free to use this thread to give more suggestions :)

**Comment by mustevenplay** [comment]
Maybe with the implementation of the VST support (whenever that is), it would be a great idea (UI-wise) to add some button to open up the plugin window, where you select, arrange and configure all your plugin chain (saving plugin chain presets would also be a great idea), and maybe, next to that button, a bypass switch for all the effects in the chain!
This idea is just so the main interface wouldn't seem so cluttered, more organised, and quite proper from an (somewhat) experienced producer/mixing engineer's point of view.

**Reply by nodeful** [reply]
I think with the fact that eqMac's UI is based on Web technologies I will have no other option besides opening the Plug-In config UI as a separate Window :) The only other option would be to stack it somewhere on the sides or bottom of eqMac which is not preferable.
